src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs

```diff
@@ -563,6 +586,9 @@ List<TelemetryEvent> ToTelemetryPackageList(List<Tuple<string, string>> packages
             {
                 // userAction.Version can be null for deleted packages.
                 actionTelemetryEvent.ComplexData["SelectedPackage"] = ToTelemetryPackage(new Tuple<string, string>(userAction.PackageId, userAction.Version?.ToNormalizedString() ?? string.Empty));
+                actionTelemetryEvent["SelectedIndex"] = selectedIndex;
+                actionTelemetryEvent["NumRecommended"] = numRecommended;
```

Comment:
```suggestion
                actionTelemetryEvent["RecommendedCount"] = numRecommended;
```

It's in line with the rest of our metrics.

Reply:
Changed to RecommendedCount